pkgs/data/icons/papirus-maia-icon-theme/default.nix

```diff
@@ -16,6 +16,12 @@ stdenv.mkDerivation rec {
     gtk3
   ];
 
+  buildInputs = [
+    breeze-icons
+    gnome-icon-theme
+    papirus-icon-theme
```

Comment:
Do they get referenced in the build?

Reply:
No. They get referenced in the installed file `share/icons/Papirus-Adapta-Maia/index.theme` (and others):

```
Inherits=Papirus,breeze,gnome,hicolor
 ```

And it expects just that those themes are installed.

Reply:
Ok. The problem is that just adding them to `buildInputs` won't make them available at runtime or installed into the user's profile.
cc @jtojnar

Reply:
Yeah, I think the inherited themes need to go to `propagatedBuildInputs`. Recently encountered this issue in https://github.com/NixOS/nixpkgs/issues/84983.